spec/features/attending_events_spec.rb

```diff
@@ -37,33 +37,34 @@
     expect(page).to have_content "Add to Google Calendar"
   end
 
-  scenario "User clicks to attend an event" do
-    sign_up
-    visit events_path
-    click_link "Blessings wedding"
-    click_link "Attend this event"
-    within ".modal-content" do
-      page.execute_script("$('#ticket_type_1').prop('checked', true)")
-      fill_in "tickets_quantity_1", with: 1
-      click_button "Submit"
-    end
+ # scenario "User clicks to attend an event" do
```

Comment:
CRITICAL: why are these commented out. is it no longer relevant? if yes, let's remove it, not commented out.